.github/workflows/nightly-merge.yml

```diff
@@ -0,0 +1,25 @@
+name: 'Nightly Merge (master to iiot)'
+
+on:
+  schedule:
+    - cron:  '0 0 * * *'
+
+jobs:
+  nightly-merge:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v1
+
+    - name: Nightly Merge
+      uses: robotology/gh-action-nightly-merge@v1.2.0
+      with:
+        stable_branch: 'master'
+        development_branch: 'iiot'
+        allow_ff: false
```

Comment:
why not allow fast forward? I think it should be true.

Reply:
Sure.  Will change together once schedule is confirmed.

Reply:
IMO fast-forward merges makes harder to follow merge history. I'd prefer merge commits for the sake of visibility

Reply:
@dmolokanov - in this case we are merging commits from master - which will have clean history (ideally only squash commits from merged PRs). So I think it's actually makes history cleaner. But nevertheless, once we start pushing changes into `iiot`, ff merge will not be possible and it will fallback to merge commits.